packages/alpha/src/api/simulators/modelsApi.ts

```diff
@@ -25,6 +27,12 @@ export class ModelsAPI extends BaseResourceAPI<SimulatorModel> {
     return this.createEndpoint(items);
   };
 
+  public aggregate = (
+    query?: SimulatorModelAggregateQuery
```

Comment:
```suggestion
    query: SimulatorModelAggregateQuery
```
query is required in our spec

Reply:
pushed up the change now